mozci/console/commands/push.py

```diff
@@ -587,6 +588,20 @@ def handle(self) -> None:
                     f"<comment>Push {push.branch}/{push.rev} was backedout and all of its failures and the ones of its children were marked as intermittent or marked as caused by another push</comment>"
                 )
 
+            for task in p.tasks:
+                for classification, note in task.classifications:
+                    if classification != "fixed by commit":
+                        continue
+
+                    try:
+                        fix_hgmo = HgRev.create(note[:12], branch=self.branch)
+                        if len(fix_hgmo.backouts) == 0:
```

Comment:
What's the use for this line since you'll continue anyway? Would it possibly raise another `PushNotFound` exception needed to trigger the warning?

Reply:
Lazy copy and paste. I'll remove it.
We could in theory check if the referenced revision contains either a backout of a bustage fix, but bustage fixes are hard to detect so the check wouldn't be precise.

Reply:
Fixed in 27446a1.